lib/l10n/app_id.arb

```diff
@@ -222,12 +222,12 @@
   "moderate": "Sedang",
   "deselect_all": "Batalkan Semua",
   "select_all": "Pilih Semua",
-  "are_you_sure": "Anda yakin?",
-  "generating_playlist": "Menghasilkan daftar putar khusus Anda...",
+  "are_you_sure": "Kamu yakin?",
+  "generating_playlist": "Menghasilkan daftar putar khususmu...",
   "selected_count_tracks": "{count} lagu yang dipilih",
-  "download_warning": "Jika Anda mengunduh semua Lagu secara massal, Anda jelas membajak Musik & menyebabkan kerusakan pada masyarakat kreatif Musik. Saya harap Anda menyadari hal ini. Selalu berusaha menghormati & mendukung kerja keras Artis",
-  "download_ip_ban_warning": "BTW, IP Anda bisa diblokir di YouTube karena permintaan unduhan yang berlebihan dari biasanya. Blokir IP berarti Anda tidak dapat menggunakan YouTube (meskipun Anda masuk) setidaknya selama 2-3 bulan dari perangkat IP tersebut. Dan Spotube tidak bertanggung jawab jika hal ini terjadi",
-  "by_clicking_accept_terms": "Dengan mengklik 'terima' Anda menyetujui ketentuan berikut:",
+  "download_warning": "Jika mengunduh semua Lagu secara massal, kamu jelas membajak Musik & menyebabkan kerusakan pada masyarakat kreatif Musik. Saya harap kamu menyadari hal ini. Selalu berusaha menghormati & mendukung kerja keras Artis",
```

Comment:
`  "download_warning": "Jika mengunduh semua Lagu secara massal, kamu jelas membajak Musik & menyebabkan kerusakan pada masyarakat kreatif Musik. Saya harap kamu menyadari hal ini. Selalu berusaha menghormati & mendukung kerja keras Artis",`

Would be better 
`  "download_warning": "Jika mengunduh terlalu banyak lagu, kamu jelas membajak Musik & menyebabkan kerusakan pada industri music. Kami harap kamu menyadari hal ini, dan selalu berusaha menghormati & mendukung kerja keras para Artis",`